From 9799144598311ce150ce2023850a16120d2f3426 Mon Sep 17 00:00:00 2001
Message-Id: <9799144598311ce150ce2023850a16120d2f3426.1427537204.git.chang-joon.lee@intel.com>
From: Hong Liu <hong.liu@intel.com>
Date: Fri, 6 Feb 2015 15:59:05 +0800
Subject: [PATCH 1/2] REVERT "REVERTME: HACK: drm/i915: Force HW reset/modeset
 at DPMS ON"

This reverts following commit, which causes MIPI black during HDMI
hotplug.

	commit 42e15165de45477d309fd9bcfd0ea878903a60cc
	Author: Daniel Leung <daniel.leung@intel.com>
	Date:   Fri Mar 7 17:20:32 2014 -0800

    	REVERTME: HACK: drm/i915: Force HW reset/modeset at DPMS ON

Issue: IMINAN-21234
Change-Id: I3d2b4505c9fd08f7f2a9233e6e96ffc0a3329702
Signed-off-by: Hong Liu <hong.liu@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 25f177f..3ff9b5e 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -5942,9 +5942,6 @@ void intel_connector_dpms(struct drm_connector *connector, int mode)
 
 	intel_display_power_get(dev_priv, PIPE_A);
 
-	if (mode == DRM_MODE_DPMS_ON)
-		intel_modeset_setup_hw_state(dev, true);
-
 	/* Only need to change hw state when actually enabled */
 	if (connector->encoder) {
 		if (mode == DRM_MODE_DPMS_ON && dev_priv->is_first_modeset) {
-- 
1.7.9.5

